joblib/test/test_hashing.py

```diff
@@ -414,18 +414,26 @@ def test_hashes_stay_the_same_with_numpy_objects():
                 rng.randint(10, size=20).astype('<i8'),
                 rng.randn(10).astype('<f4')
             ]
-        }
+        },
+        # Non regression cases for https://github.com/joblib/joblib/issues/308.
+        # Generated with joblib 0.9.4.
+        np.ones((10, 10)),
```

Comment:
maybe something else than a constant array here, so that there is a difference between C and Fortran order.